controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller_unit_test.go

```diff
@@ -155,6 +155,9 @@ func unitTestsReconcile() {
 						Status: corev1.ConditionTrue,
 					},
 				}
+				labels := make(map[string]string)
+				labels[utils.ContentLibraryServiceTypeLabelKey] = ""
+				cclItem.ObjectMeta.Labels = labels
```

Comment:
This label setup is irrelevant to the statement from the "When" block above. Can you instead add the label in the `DummyClusterContentLibraryItem` function and update all positive test cases to check the expected labels from cvmi?